.dockerignore

```diff
@@ -1,5 +1,6 @@
 npm-debug.log
 node_modules
 build
+config/default.yaml
 configs
```

Comment:
What is the configs directory? Also, why not just ignore the entire config directory?

Reply:
1. configs is the folder Jenkins will clone from configs repo.
2. `custom-environment-variables.yaml` is the file that we let the API know which field in `default.yaml` should be overwritten by env vars and we don't wanna ignore these files.